test/shared/Env.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity 0.8.15;
```

Comment:
minor: let's keep `^0.8.0` for all contracts not intended for production deployment: tests, scripts, interfaces, etc...